sw/device/lib/crt/crt.S

```diff
@@ -0,0 +1,134 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// CRT library
+//
+// Utility functions written in assembly that can be used before the C
+// runtime has been initialized. These functions should not be used once
+// the C runtime has been initialized.
```

Comment:
Nit: Maybe put this in `/**/` comments for consistency?

Reply:
Done